pyctcdecode/alphabet.py

```diff
@@ -105,20 +125,20 @@ def labels(self) -> List[str]:
         return self._labels[:]  # this is a copy
 
     @classmethod
-    def build_alphabet(
-        cls, label_list: List[str], ctc_token_idx: Optional[int] = None
-    ) -> "Alphabet":
-        """Make a non-BPE alphabet."""
-        formatted_alphabet_list = _normalize_alphabet(label_list, ctc_token_idx)
-        return cls(formatted_alphabet_list, False)
+    def build_alphabet(cls, labels: List[str]) -> "Alphabet":
+        """Make an alphabet from labels in standardized format for decoder."""
+        is_bpe = _check_if_bpe(labels)
+        _verify_alphabet(labels, is_bpe)
+        if is_bpe:
+            normalized_labels = _normalize_bpe_alphabet(labels)
+        else:
+            normalized_labels = _normalize_regular_alphabet(labels)
+        return cls(normalized_labels, is_bpe)
 
-    @classmethod
-    def build_bpe_alphabet(
-        cls,
-        label_list: List[str],
-        unk_token_idx: Optional[int] = None,
-        ctc_token_idx: Optional[int] = None,
-    ) -> "Alphabet":
-        """Make a BPE alphabet."""
-        formatted_label_list = _normalize_bpe_alphabet(label_list, unk_token_idx, ctc_token_idx)
-        return cls(formatted_label_list, True)
+
+def verify_alphabet_coverage(alphabet: Alphabet, unigrams: Collection[str]) -> None:
+    """Verify if alphabet covers a given unigrams."""
+    label_chars = set(alphabet.labels)
+    unigram_sample_chars = set("".join(unigrams))
```

Comment:
mildly surprising, why can't we just call `set`?

Reply:
we need the set of characters, not words. maybe i'm missing something

Reply:
right right right, my thinko